### ISSUE TITLE
Backend: Use kubectl proxy package

### ISSUE THREAD

**Issue by yolossn** [headlamp-k8s/headlamp#1166]
This issue is for documentation of the efforts that was part of https://github.com/headlamp-k8s/headlamp/pull/1152 to use the [proxy](https://github.com/kubernetes/kubectl/blob/master/pkg/proxy/proxy_server.go) package in kubectl. At present the kubectl proxy doesn't properly support feature like attach, exec via websocket https://github.com/kubernetes/kubernetes/issues/89163. Once the issue is solved in the upstream then headlamp can shift to using the proxy package from kubectl

**Comment by illume** [comment]
@yolossn is this issue still valid? Looks like the issue is fixed upstream now.

**Comment by yolossn** [comment]
Yes it is valid, thanks for bringing the upstream change to my attention. Ill take a look.
